theme/admin.py

```diff
@@ -33,11 +33,19 @@ class GalleryAdmin(DisplayableAdmin):
     list_display = ('title', 'status')
     inlines = [GalleryImageInline]
 
+class SponsorTypeAdmin(DisplayableAdmin):
+    def get_model_perms(self, request):
```

Comment:
Hiding sponsor type from admin menu, it looks more suitable when inline with "Sponsor type" in sponsor admin page.